apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -0,0 +1,147 @@
+    const country = 'ru'
+    const name = faker.company.companyName ()
+    const description = faker.company.catchPhrase ()
+    const meta = {
+        dv: 1, inn: '6670428515', kpp: '667001001', city: faker.address.city (), zipCode: faker.address.zipCode (),
+        street: faker.address.streetName (), number: faker.address.secondaryAddress (),
+        county: faker.address.county (),
+    }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        country, name, description, meta,
+        ...extraAttrs,
+    }
+    const obj = await Organization.create(client, attrs)
+    return [obj, attrs]
+}
+
+async function updateTestOrganization (client, id, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!id) throw new Error('no id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        ...extraAttrs,
+    }
+    const obj = await Organization.update(client, id, attrs)
+    return [obj, attrs]
+}
+
+async function createTestOrganizationEmployee (client, organization, user, role, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!organization || !organization.id) throw new Error('no organization.id')
+    if (!user || !user.id) throw new Error('no user.id')
+    if (!role || !role.id) throw new Error('no role.id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        email: faker.internet.email(),
+        organization: { connect: { id: organization.id } },
+        user: { connect: { id: user.id } },
+        role: { connect: { id: role.id } },
+        ...extraAttrs,
+    }
+    const obj = await OrganizationEmployee.create(client, attrs)
+    return [obj, attrs]
+}
+
+async function updateTestOrganizationEmployee (client, id, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!id) throw new Error('no id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        ...extraAttrs,
+    }
+    const obj = await OrganizationEmployee.update(client, id, attrs)
+    return [obj, attrs]
+}
+
+async function createTestOrganizationEmployeeRole (client, organization, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!organization || !organization.id) throw new Error('no organization.id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        name: faker.random.alphaNumeric(8),
+        organization: { connect: { id: organization.id } },
+        ...extraAttrs,
+    }
+    const obj = await OrganizationEmployeeRole.create(client, attrs)
+    return [obj, attrs]
+}
+
+/**
+ * Simplifies creating series of instances
+ */
+async function createTestOrganizationEmployeeFullChain () {
```

Comment:
(optional)

You can also check makeClientWithRegisteredOrganization

Is this function used anywhere?

Reply:
Yes, in 4 places. It incapsulates the same setup in different test cases.

Reply:
Changed in `bae61c6`